subsys/bluetooth/host/hci_core.c

```diff
@@ -4112,6 +4218,25 @@ static int le_init(void)
 	}
 #endif
 
+#if defined(CONFIG_BT_WHITELIST)
+	struct bt_hci_rp_le_read_wl_size *rp;
+
+	err = bt_hci_cmd_send_sync(BT_HCI_OP_LE_READ_WL_SIZE, NULL,
+				   &rsp);
+
+	if (err) {
+		return err;
+	}
+
+	rp = (void *)rsp->data;
```

Comment:
why cast to `void *` instead of `struct bt_hci_rp_le_read_wl_size *`?

Reply:
Absolutely no reason (I just duplicated the rl_size read), will change :)